examples/basic_mysql_connection/index.js

```diff
@@ -2,19 +2,33 @@ var fs = require('fs');
 var mysql = require('mysql');
 
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
-
+  
 var connection = mysql.createConnection({
-	host: config.host,
-	user: config.user,
-	password: config.password,
-	port: config.port,
+ 	  host: config.host,
+  	user: config.user,
+  	password: config.password,
+  	port: config.port,
     database: config.database
 });
-
-connection.connect();
-
-connection.query('SELECT * FROM todos', function (error, results, fields) {
-    console.log(results);
-});
-
-connection.end();
\ No newline at end of file
+  
+   
+const sqlStatments=[
+        `INSERT INTO todos (Done, Name, StatusId, Due) VALUES (1, 'Review SCSS', 1,'2017-08-17 00:00:00')`,
```

Comment:
Great job using the correct capitalisation (thanks for using `Done` instead of `done` or `DONE`)! 😄